sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_hub_operation.py

```diff
@@ -81,7 +82,7 @@ def list(self, *, scope: str = Scope.RESOURCE_GROUP) -> Iterable[WorkspaceHub]:
     # @monitor_with_activity(logger, "Hub.Get", ActivityType.PUBLICAPI)
     @distributed_trace
     # pylint: disable=arguments-renamed
-    def get(self, name: str, **kwargs: Dict) -> WorkspaceHub:
+    def get(self, *, name: str, **kwargs: Dict) -> WorkspaceHub:
         """Get a Workspace WorkspaceHub by name.
 
         :param name: Name of the WorkspaceHub.
```

Comment:
Needs to be

```
        :keyword name: Name of the WorkspaceHub.
        :paramtype name: str
        :return: The WorkspaceHub with the provided name.
        :rtype: WorkspaceHub
```

Reply:
Reverted since get can't be keyword only due to the wierd class inheritance of the operations class that would require a refactor to fix.